client/statsdb.js

```diff
@@ -51,14 +53,15 @@ class StatsDB {
   }
 
   // / Takes the account which has executed a transaction.
-  async updateExecuted(account) {
+  async updateExecuted(account, timeBounty, fee, gas) {
```

Comment:
You don't need to pass `fee` nor `gas` into this function. At execution, you only care about the `bounty` since the gas will come back to the timenode no matter what. UNLESS THE TRANSACTION IS REVERTED. You need to handle the case that `receipt.status === '0x00'` or in recent versions of the libraries `receipt.status === false`. Only if these two prior condition equal `true` do you need to calculate the gas expenditure as a `cost`.